crates/extension_host/src/wasm_host/wit/since_v0_2_0.rs

```diff
@@ -497,6 +497,52 @@ impl nodejs::Host for WasmState {
     }
 }
 
+impl ruby::Host for WasmState {
```

Comment:
This definitely should be in the next version of the Extension API, but this part of Zed is still a mystery to me. I tried the simplest approach by copying and pasting but failed.